release_docs.sh

```diff
@@ -22,6 +22,7 @@ then
   echo "<meta http-equiv='refresh' content='0; url=/parse-server/api/${DEST}'>" > "docs/api/index.html"
 fi
 
+npm run definitions
```

Comment:
ok to leave, but not sure we need this, right.  Since the output is committed.

Reply:
I added it here since people have been manually changing the file.